integration-tests/smoke/automation_test.go

```diff
@@ -1005,27 +1016,34 @@ func setupAutomationTestDocker(
 	clNodeConfig.P2P.V2.AnnounceAddresses = &[]string{"0.0.0.0:6690"}
 	clNodeConfig.P2P.V2.ListenAddresses = &[]string{"0.0.0.0:6690"}
 
-	secretsConfig := `
-	[Mercury.Credentials.cred1]
-	LegacyURL = 'http://localhost:53299'
-	URL = 'http://localhost:53299'
-	Username = 'node'
-	Password = 'nodepass'
-	`
-
 	//launch the environment
 	env, err := test_env.NewCLTestEnvBuilder().
 		WithTestLogger(t).
 		WithGeth().
 		WithMockServer(1).
-		WithCLNodes(5).
-		WithCLNodeConfig(clNodeConfig).
-		WithSecretsConfig(secretsConfig).
+		//WithCLNodes(5).
+		//WithCLNodeConfig(clNodeConfig).
+		//WithSecretsConfig(secretsConfig).
 		WithFunding(big.NewFloat(.5)).
 		Build()
 	require.NoError(t, err, "Error deploying test environment")
+
+	secretsConfig := `
+	[Mercury.Credentials.cred1]
+	LegacyURL = '%s'
+	URL = '%s'
+	Username = 'node'
+	Password = 'nodepass'
+	`
+	secretsConfig = fmt.Sprintf(secretsConfig, env.MockServer.InternalEndpoint, env.MockServer.InternalEndpoint)
+
+	err = env.StartClNodes(clNodeConfig, 5, secretsConfig)
```

Comment:
@AnieeG I tried above, 1) remove nodeConfig, secretsConfig, nodeCount before build() 2) construct the secrets config with the right Endpoint 3) StartClNodes with nodeConfig, count and secretsConfig.

I got the following error, I believe it is related to the testing env setup, but I couldnt make much sense from it. I used this command to run in my local: `make test_smoke_product product="automation" args="-timeout 20m -test.run TestAutomationBasic/registry_2_1_logtrigger"`

```
log.go:29: 14:05:36.10 INF 🚧 Waiting for container id c9990ad9fc22 image: chainlink:develop. Waiting for: &{timeout:0x140028bc790 Port:6688/tcp Path:/health StatusCodeMatcher:0x10509e950 ResponseMatcher:0x1058e2990 UseTLS:false AllowInsecure:false TLSConfig:<nil> Method:GET Body:<nil> PollInterval:1s UserInfo:}
    log.go:29: 14:05:37.00 DBG Received Header Hash=0xc3c37f4236adea7574c1faa9c507a6f095b47adcaaed538d28b8a7e1ecfbbdae NetworkName=geth Node=0 Number=24
    log.go:29: 14:05:38.00 DBG Received Header Hash=0x9dc14e40692c24a928cc7d63b19528ebd264a46ca5104a257fcd24297d5dfb53 NetworkName=geth Node=0 Number=25
    log.go:29: 14:05:39.01 DBG Received Header Hash=0xbdd9cb06afb64eb0c4981198ba68693a07c1d223b7102054bde31fbe947cdc28 NetworkName=geth Node=0 Number=26
    log.go:29: 14:05:39.19 INF Started Chainlink Node container clEndpoint=http://localhost:52860 clInternalIP=192.168.112.13 containerName=cl-node-882e0ebc
    log.go:29: 14:05:39.39 DBG Calculated ETH for Chainlink Operations Gas Limit per Operation=510000 Number of Operations=1000 Total (ETH)=0.01584914811 Value per Operation (ETH)=1.584914811e-05
    log.go:29: 14:05:39.39 INF Reading ETH Keys Node URL=http://localhost:52839
    log.go:29: 14:05:39.39 WRN Found no ETH Keys on the node Node URL=http://localhost:52839
--- FAIL: TestAutomationBasic (35.82s)
    --- FAIL: TestAutomationBasic/registry_2_1_logtrigger (35.82s)
panic: runtime error: index out of range [0] with length 0 [recovered]
        panic: runtime error: index out of range [0] with length 0

goroutine 256 [running]:
testing.tRunner.func1.2({0x1072d9140, 0x14023bceac8})
        /opt/homebrew/opt/go/libexec/src/testing/testing.go:1526 +0x1c8
testing.tRunner.func1()
        /opt/homebrew/opt/go/libexec/src/testing/testing.go:1529 +0x384
panic({0x1072d9140, 0x14023bceac8})
        /opt/homebrew/opt/go/libexec/src/runtime/panic.go:884 +0x204
github.com/smartcontractkit/chainlink/integration-tests/client.(*ChainlinkClient).PrimaryEthAddress(0x14001d82370)
        /Users/leishi/chainlink/integration-tests/client/chainlink.go:545 +0xb8
github.com/smartcontractkit/chainlink/integration-tests/actions.FundChainlinkNodesLocal({0x14023aa9d48, 0x4, 0x0?}, {0x107663e08, 0x140011e97a0}, 0x1?)
        /Users/leishi/chainlink/integration-tests/actions/ocr_helpers_local.go:33 +0x70
command-line-arguments.setupAutomationTestDocker(0x140026e2ea0, {0x0?, 0x0?}, 0x5, {0xbebc200, 0x0, 0x140026aa480, 0x2625a0, 0x140026aa4a0, 0x1, ...}, ...)
        /Users/leishi/chainlink/integration-tests/smoke/automation_test.go:1051 +0x754
command-line-arguments.SetupAutomationBasic.func1(0x140026e2ea0)
        /Users/leishi/chainlink/integration-tests/smoke/automation_test.go:116 +0x248
testing.tRunner(0x140026e2ea0, 0x140026aa600)
        /opt/homebrew/opt/go/libexec/src/testing/testing.go:1576 +0x10c
created by testing.(*T).Run
        /opt/homebrew/opt/go/libexec/src/testing/testing.go:1629 +0x368
FAIL    command-line-arguments  37.188s
FAIL
make: *** [test_smoke_product] Error 1

```